common.yml

```diff
@@ -517,16 +517,14 @@ functions:
 
           echo -n "$MONGOD_URI" > MONGOD_URI
 
-  "check third-party notices":
-    - command: shell.exec
-      type: test
-      params:
-        working_dir: src/github.com/mongodb/mongo-tools
-        script: |
-          set -e
-          set -x
-          ./etc/generate-notices.pl > ./THIRD-PARTY-NOTICES
-          git diff --exit-code
+  "check sbom lite":
+    command: shell.exec
+    type: test
+    params:
+      working_dir: src/github.com/mongodb/mongo-tools
+      script: |
+        ${_set_shell_env}
+        ./scripts/regenerate-and-diff-sbom.sh
```

Comment:
I remember we wanted to get rid of all the shell script and in favor of Go code?

Reply:
If I rewrote this in Go all it would do is execute exactly the same commands. I don't think that would improve things much. I think the case for "don't use shell" is more for complex programs like the code we have in Mongosync to download Server versions.

Reply:
I think the purpose of "not use shell" is for better cross-platform support. Since this is a release script we probably can only run it on Linux, I agree it's probably fine to write in shell.